### ISSUE TITLE
Screen reader unable to read list of changed files.

### ISSUE THREAD

**Issue by Leos-Khai** [desktop/desktop#17048]
### The problem

With the current version, I am unable to read the list of changed / modified files with NVDA screen reader. Both focus and brouze mode is the same.

### Release version

Version 3.2.6 (x64)

### Operating system

Windows 11

### Steps to reproduce the behavior

1. Open github desktop with NVDA running.
2. Change a file in your code base.
3. Go to the list of changed file with your screen reader.
4. It will not read.

**Comment by sergiou87** [comment]
Hi @jamestoh-leos ! 👋 

I cannot test it right now with NVDA, but at least macOS' VoiceOver works as expected, and I'd say all the accessibility attributes are properly set to the elements of that list. We're _right now_ doing **a ton** of work improving the accessibility of the app, so you can expect many changes in every new version.

While I will test this with NVDA as soon as possible, may I suggest trying out the latest beta version? You can download it from https://desktop.github.com/beta/

**Comment by Leos-Khai** [comment]
Thanks @sergiou87 , The beta version is properly labled and I am able to read them.

**Comment by sergiou87** [comment]
Awesome! Thank you for letting us know 😄 It will take some time for those fixes to get to production builds, since we're in "Summer mode" with the team taking some time off. Thank you for your patience 🙇‍♂️
